main.js

```diff
@@ -302,23 +310,24 @@ function setBoxesAndMoves() {
 
 function newChallenger() {
   if (!currentGame.currentZodiac.length) {
+    debugger
```

Comment:
This will be removed eventually, I hope.

Reply:
Good catch! I totally missed this, thank you for noticing